src/vs/workbench/common/contextkeys.ts

```diff
@@ -55,6 +55,7 @@ export const ActiveEditorReadonlyContext = new RawContextKey<boolean>('activeEdi
 export const ActiveEditorCanToggleReadonlyContext = new RawContextKey<boolean>('activeEditorCanToggleReadonly', true, localize('activeEditorCanToggleReadonly', "Whether the active editor can toggle between being read-only or writeable"));
 export const ActiveEditorCanRevertContext = new RawContextKey<boolean>('activeEditorCanRevert', false, localize('activeEditorCanRevert', "Whether the active editor can revert"));
 export const ActiveEditorCanSplitInGroupContext = new RawContextKey<boolean>('activeEditorCanSplitInGroup', true);
+export const ActiveEditorProblemsContext = new RawContextKey<boolean>('activeEditorProblemsContext', true);
```

Comment:
I am not sure the need for this context key? You can use auto-generated context keys if you want to make an action adjust based on a setting, for example `ContextKeyExpr.has('config.diffEditor.hideUnchangedRegions.enabled')`